src/components/MoviesList/MoviesList.tsx

```diff
@@ -1,6 +1,27 @@
 import React from 'react';
 import './MoviesList.scss';
+import { MovieCard } from '../MovieCard';
+import { Movie } from '../../types/Movie';
 
-export const MoviesList: React.FC = () => (
-  <>Put the list here</>
-);
+type Props = {
+  movies: Movie[];
+};
+
+export const MoviesList: React.FC<Props> = ({ movies }) => {
+  const content
+  = movies.length > 0 ? (
```

Comment:
Just don't render the MovieList component if you don't have any movies

Reply:
Done